lib/src/clojure_lsp/kondo.clj

```diff
@@ -121,14 +121,10 @@
       (if (:api? @db)
         (do
           (logger/info (format "Starting to lint whole project files..."))
-          (shared/logging-time
-            "Linting whole project files took %s secs"
-            (f.diagnostic/lint-project-diagnostics! new-analysis kondo-ctx db)))
+          (f.diagnostic/lint-project-diagnostics! new-analysis kondo-ctx db))
         (when (settings/get db [:lint-project-files-after-startup?] true)
           (async/go
```

Comment:
which the huge performance improvement, I'll consider remove the async of this function, making sync for both cli and editor would result in a way easier code to understand and probably fix some concurrencies issues